backport-pull-request/README.md

```diff
@@ -0,0 +1,80 @@
+# Greenbone Pull Request Backport Action
+
+GitHub Action to backport pull requests.
+
+To run this action you need to add the following code to your workflow file
+(for example `.github/workflows/backport.yml`):
+
+```yml
+name: Backport
+
+on:
+  pull_request:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    name: Backport Pull Requests
+    runs-on: ubuntu-latest
+    steps:
+        - name: Backport Pull Request
+          uses: bjoernricks/pull-request-backport-action@main
```

Comment:
```suggestion
          uses: greenbone/actions/pull-request-backport-action@v1
```

Reply:
Should be `greenbone/actions/backport-pull-request@v1`. thanks